core/locale/en.inc.php

```diff
@@ -0,0 +1,570 @@
+<?php
+$arrLang = array();
+
+$arrLang['alert_admin_email_wrong'] = "Admin email has wrong format! Please re-enter.";
+$arrLang['alert_min_version_db'] = "This program requires at least version _DB_VERSION_ of _DB_ installed (current version is _DB_CURR_VERSION_). You cannot proceed the installation.";
+$arrLang['alert_min_version_php'] = "This program requires at least version _PHP_VERSION_ of PHP installed (current version is _PHP_CURR_VERSION_). You cannot proceed the installation.";
```

Comment:
Something tells me this should say "You cannot proceed with the installation". If that's just the English grad in me needing to shut up, also let me know.